modules/recommended-requirements-central-services.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * installing/acs-recommended-requirements.adoc
+:_content-type: CONCEPT
+[id="recommended-requirements-central-services_{context}"]
+= Central services (self-managed)
+
+Central services contain the following components:
+
+* Central
+* Scanner
+
+[NOTE]
+====
+If you are using {rh-rhacscs-first}, you do not need to review the requirements for Central services, because they are managed by Red Hat. You only need to look at the requirements for secured cluster services.
+====
+
+[NOTE]
+====
+For default resource requirements for the scanner, take a look at the *default resource requirements* page.
```

Comment:
I have moved this note under bullet point. And I have moved the previous note under the title. Because it is actually relevant for the whole Central services section, not only "Central".

I did the same for default requirements.